bin/index.js

```diff
@@ -108,7 +108,9 @@ function initCommand(options) {
 
   const copySpinner = createSpinner("Creating server files...").start();
   try {
-    fs.copySync(templatePath, destinationPath);
+    await fs.copy(templatePath, targetDir, {
+      filter: (src) => !src.includes("node_modules"), // Skiping node_modules to optimize
+    });
```

Comment:
`node_modules` will never be there inside the templates folder!!
This change is not needed. It will not optimize.

Reply:
ok